.azure-pipelines/client.yml

```diff
@@ -99,6 +102,7 @@ jobs:
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
           AZ_CONFIG_CONNECTION: $(AzConfigConnectionString)
+          azure-service-bus-dotnet/connectionstring: $(ServiceBusConnectionString)
```

Comment:
Is `/` and allowed character in an env variable name?

Reply:
It is, though it was an odd choice.   This was unchanged from the Service bus SDK on import.  See: [https://github.com/Azure/azure-service-bus-dotnet](https://github.com/Azure/azure-service-bus-dotnet)

Reply:
I think we should rename this eventually, but the current name is verified to work on all OS.